### PR TITLE
[EDU-3650]refactor: remove initiator type from guides

### PR DESCRIPTION
**Pull request by gabriel-azion**
Related issue: 
[EDU-3650](https://aziontech.atlassian.net/browse/EDU-3650)

Changes

- Remove mentions to initiator type (edge functions)

[EDU-3650]: https://aziontech.atlassian.net/browse/EDU-3650?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/pt-br/pages/guias/edge-functions-firewall/index.mdx [hunk 1]
```diff
@@ -30,16 +30,15 @@ Para usar functions no seu Edge Firewall, você deve seguir dois passos:
...
 Agora, sua edge function pode ser usada em configurações do edge firewall.
 
 ---
 
 ## Configurando uma edge function no Edge Firewall
 
-Depois de criar uma edge function com o `Initiator type` definido como Edge Firewall, você precisa configurar o firewall para implementá-la.
+Depois de criar uma edge function, você precisa configurar o firewall para implementá-la.
 
 ### Criando um Rules Set
```
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/guias/edge-functions-firewall/index.mdx:43]:
```suggestion
### Criando um rule set
```

File: src/content/docs/en/pages/guides/edge-functions-firewall/index.mdx [hunk 2]
```diff
@@ -30,16 +30,15 @@ To use functions in your Edge Firewall, you have to go through two steps:
...
 Now, your edge function can be used on an edge firewall configuration.
 
 ---
 
 ## Configuring your edge function on the Edge Firewall 
 
-After creating an edge function with the `Initiator type` set as Edge Firewall, you need to configure the edge firewall to use it.
+After creating an edge function, you need to configure the edge firewall to use it.
 
 ### 2.1 Creating a Rules Set
```
Comment (bru-andrade) [inline src/content/docs/en/pages/guides/edge-functions-firewall/index.mdx:43]:
```suggestion
### 2.1 Creating a rule set
```

**Review by bru-andrade — APPROVED** [review]
Apenas duas sugestões ✨

**Review by MarianaAguilera — APPROVED** [review]
Approved with @bru-andrade 's comments

**Review by hannah-sk — APPROVED** [review]

**Pull request merged**